application/contracts/pokemon_repository.go

```diff
@@ -0,0 +1,11 @@
+package contracts
+
+import "github.com/FernandoGal25/academy-go-q42021/domain/model"
+
+// Interface of pokemon repository, defines the methods of the repository
```

Comment:
Documentation comments should start with the name of the element. Please fix this for all other occurrences of this issue.
```suggestion
// PokemonRepository interface of pokemon repository, defines the methods of the repository
```